requirements.txt

```
@@ -1,3 +1,4 @@
 colorama==0.4.4
 chess==1.5.0
 requests==2.20.0
+lichess>=0.1.9
```

I had a problem installing this and had to change it to:
```suggestion
colorama==0.4.4
chess==1.5.0
requests==2.27.1
lichess>=0.1.9
```

This was the error
```
ERROR: Cannot install -r requirements.txt (line 4) and requests==2.20.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested requests==2.20.0
    lichess 0.1.9 depends on requests==2.27.1
```